Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +254,224 @@ edgebridgetutorialapp://
 ### 2. Connect the app to the Assurance session  
 To connect the tutorial app to the Assurance session, see the instructions on [connecting the app to the Assurance session](https://github.com/adobe/aepsdk-edge-ios/blob/main/Documentation/Tutorials/edge-send-event-tutorial.md#2-connect-to-the-app).
```

Comment:
What I mean is to update the instructions here instead of referencing to the Edge tutorial. I saw you added the Assurance view to the tutorial application so users can just paste the session ID into the app, but the tutorial still has users use a deep link to connect to Assurance. I thought your intention was to also update these steps to use the in-app view to connect to Assurance?